File: event-streams-java/event-sender/src/main/java/com/scientiamobile/wurflmicroservice/eventsender/EventData.java
package com.scientiamobile.wurflmicroservice.eventsender;

import com.google.gson.annotations.SerializedName;

/**
 * Event data class is the model for the data arriving from the JSON event stream. The only data which is added
 * during this phase if the timestamp (time at which JSON event is sent to output in nanoseconds).
 */
class EventData {

    @SerializedName(value = "X-Ucbrowser-Ua")
    private String ucBrowserUa;
    @SerializedName(value = "X-Ucbrowser-Device-Ua")
    private String ucBrowserDeviceUa;
    @SerializedName(value = "Accept-Language")
    private String acceptLanguage;
    @SerializedName(value = "Accept-Encoding")
    private String acceptEncoding;
    @SerializedName(value = "Accept-Charset")
    private String acceptCharset;
    @SerializedName(value = "X-Forwarded-For")
    private String xForwardedFor;
    @SerializedName(value = "Accept")
    private String accept;
    @SerializedName(value = "User-Agent")
    private String userAgent;
    @SerializedName(value = "Video_id")
    private String videoID;
    @SerializedName(value = "Clientip")
    private String clientIP;
    @SerializedName(value = "Event")
    private String event;
    @SerializedName("X-Operamini-Features")
    private String xOperaminiFeatures;
    @SerializedName("X-Operamini-Phone")
    private String xOperaminiPhone;
    @SerializedName("X-Operamini-Phone-Ua")
    private String xOperaminiPhoneUa;
    @SerializedName("Save-Data")
    private String saveData;
    @SerializedName("X-Clacks-Overhead")
    private String xClacksOverhead;
    @SerializedName("Sec-Fetch-Site")
    private String secFetchSite;
    @SerializedName("Sec-Fetch-Mode")
    private String secFetchMode;
    @SerializedName("Request-Context")
    private String requestContext;
    @SerializedName("Correlation-Context")
    private String correlationContext;
    @SerializedName("X-Ms-Request-Root-Id")
    private String xMsRequestRootID;
    @SerializedName("X-Ms-Request-Id")
    private String xMsRequestID;

    // timestamp in nanoseconds, when the event is sent to output (not in the original stream)
    @SerializedName(value = "timestamp")
    private long timestamp;

    public void setTimestamp(long timestamp) {
        this.timestamp = timestamp;
    }
}

File: event-streams-dotnet/event-sender/EventData.cs
using Newtonsoft.Json;
using System;

namespace event_sender
{
    class EventData
    {
        [JsonProperty("X-Ucbrowser-Ua")]
        private String ucBrowserUa;

        [JsonProperty("X-Ucbrowser-Device-Ua")]
        private String ucBrowserDeviceUa;

        [JsonProperty("Accept-Language")]
        private String acceptLanguage;

        [JsonProperty("Accept-Encoding")]
        private String acceptEncoding;

        [JsonProperty("Accept-Charset")]
        private String acceptCharset;

        [JsonProperty("X-Forwarded-For")]
        private String xForwardedFor;
        
        private String accept;

        [JsonProperty("User-Agent")]
        private String userAgent;

        [JsonProperty("Video_id")]
        private String videoID;

        [JsonProperty("Clientip")]
        private String clientIP;

        [JsonProperty("Event")]
        private String eventResult;

        [JsonProperty("X-Operamini-Features")]
        private String xOperaminiFeatures;

        [JsonProperty("X-Operamini-Phone")]
        private String xOperaminiPhone;

        [JsonProperty("X-Operamini-Phone-Ua")]
        private String xOperaminiPhoneUa;

        [JsonProperty("Save-Data")]
        private String saveData;

        [JsonProperty("X-Clacks-Overhead")]
        private String xClacksOverhead;

        [JsonProperty("Sec-Fetch-Site")]
        private String secFetchSite;

        [JsonProperty("Sec-Fetch-Mode")]
        private String secFetchMode;

        [JsonProperty("Request-Context")]
        private String requestContext;

        [JsonProperty("Correlation-Context")]
        private String correlationContext;

        [JsonProperty("X-Ms-Request-Root-Id")]
        private String xMsRequestRootID;

        [JsonProperty("X-Ms-Request-Id")]
        private String xMsRequestID;

        // timestamp in nanoseconds, when the event is sent to output (not in the original stream)
        [JsonProperty("timestamp")]
        private long timestamp;

        public void SetTimestamp(long timestamp)
        {
            this.timestamp = timestamp;
        }
    }
}